clients/fossid-webapp/src/main/kotlin/model/status/DownloadStatus.kt

```diff
@@ -22,6 +22,7 @@ package org.ossreviewtoolkit.clients.fossid.model.status
 import com.fasterxml.jackson.annotation.JsonProperty
 
 enum class DownloadStatus {
+    FAILED,
```

Comment:
Commit message: Please add a short description that this prevents crashes if the FossId download failed.

Reply:
But this change does not prevent/change crash handling :)

Before: enum cannot be mapped, hence exception, catched by `scanPackages`.
After: enum is mapped but when thows an exception, catched by `scanPackages`.

Reply:
I put an another sentence as the suggested one. Please review.